src/Change.php

```diff
@@ -587,22 +587,57 @@ public function rawSearchOptions()
             ]
         ];
 
+        $tab[] = [
+            'id'                 => 'change',
+            'name'               => __('Changes')
```

Comment:
Should be "problem" instead of change here, shouln't it?
In any way, you should use item's `getTypeName()`.